internal/test/e2e/nethttp/traces.json

```diff
@@ -68,6 +68,30 @@
                   "value": {
                     "intValue": "200"
                   }
+                },
+                {
+                  "key": "net.host.name",
+                  "value": {
+                    "stringValue": "localhost:8080"
+                  }
+                },
+                {
+                  "key": "net.protocol.name",
+                  "value": {
+                    "stringValue": "HTTP/1.1"
+                  }
+                },
+                {
+                  "key": "net.peer.name",
+                  "value": {
+                    "stringValue": "::1"
+                  }
+                },
+                {
+                  "key": "net.peer.port",
```

Comment:
These need to be updated to follow the [`v1.23` semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.23.0/docs/http/http-spans.md#common-attributes).

Reply:
`go.opentelemetry.io/otel/semconv/v1.21.0` is the latest one we can import now, right?
I couldn't find `v1.23` in that repo.

Reply:
Ah, that looks correct. We need to update there first. Resolving.